### PR TITLE
Added more Picross games

### PR DESCRIPTION
**Pull request by SonofUgly**
And updated the titleIDs, and tested PC specs, on Picross e series games.

Pokemon Picross and My Nintendo Picross: The Legend of Zelda: Twilight Princess have the same titleID for all regions (a few eShop games do this), so I've set the region to 'ALL', hopefully that'll be added to the site as a valid region.

Also updated the wiki.

### CODE REVIEW

**Comment by Schplee** [comment]
LGTM, thanks for updating your other entries. We'll merge this PR once we've added support for 'ALL'.

File: Picross-e2/game.dat [hunk 1]
```diff
@@ -13,11 +18,12 @@ title = "0004000000084600"
 region = "JPN"
 release_date = "2011-12-28"
 
+
```
Comment (Leo626) [inline Picross-e2/game.dat]: Extra line?

File: Picross-e3/game.dat [hunk 2]
```diff
@@ -13,11 +18,12 @@ title = "00040000000EEA00"
 region = "JPN"
 release_date = "2013-06-12"
 
+
```
Comment (Leo626) [inline Picross-e3/game.dat]: Same here.

File: Picross-e2/game.dat (hunk 1 above)
Comment (SonofUgly) [inline Picross-e2/game.dat]: Whoops, fixed.

**Comment by wwylele** [comment]
I don't like that stretched "boxart" at all. If it is because our constraint on the size, either you should add blank strip to keep the aspect ratio, or we should change the rule.

**Comment by SonofUgly** [comment]
@wwylele - Yeah I'm also not a fan of changing the aspect ratio for eShop title's boxart, but that's what the contributing guide said to do. Having them at 328x197 with white padding to 328x300 would look better IMO.

**Comment by Clector** [comment]
@wwylele Yes, I agree I don"t like the look of the stretched "boxart" too.
Changing the rule or let adding blank strips to keep the aspect ratio in digital only games should be a good idea to don"t make look stretched in the Citra website.

**Comment by SonofUgly** [comment]
Done.

**Comment by j-selby** [comment]
Your Picross 5e image is the incorrect resolution

**Comment by j-selby** [comment]
Hey - we have made a few changes to the main repo.

You are going to need to rebase your changes - could you do this, when you get a chance?

Thanks!

**Comment by j-selby** [comment]
Cool, looks great - thanks for the quick response.

We are waiting for the website to update to the new style of boxart, and then your PR will be merged.

**Pull request merged**